src/main/kotlin/app/revanced/patches/tiktok/misc/settings/patch/SettingsPatch.kt

```diff
@@ -8,104 +8,110 @@ import app.revanced.patcher.data.BytecodeContext
 import app.revanced.patcher.patch.annotations.DependsOn
 import app.revanced.patcher.patch.annotations.Patch
+import app.revanced.patcher.util.proxy.mutableTypes.MutableMethod
 import app.revanced.patches.tiktok.misc.integrations.patch.IntegrationsPatch
 import app.revanced.patches.tiktok.misc.settings.annotations.SettingsCompatibility
-import app.revanced.patches.tiktok.misc.settings.fingerprints.AboutViewFingerprint
+import app.revanced.patches.tiktok.misc.settings.fingerprints.AboutPageFingerprint
 import app.revanced.patches.tiktok.misc.settings.fingerprints.AdPersonalizationActivityOnCreateFingerprint
 import app.revanced.patches.tiktok.misc.settings.fingerprints.SettingsOnViewCreatedFingerprint
 import org.jf.dexlib2.Opcode
+import org.jf.dexlib2.builder.instruction.BuilderInstruction21c
 import org.jf.dexlib2.iface.instruction.OneRegisterInstruction
 import org.jf.dexlib2.iface.instruction.ReferenceInstruction
+import org.jf.dexlib2.iface.instruction.WideLiteralInstruction
 import org.jf.dexlib2.iface.instruction.formats.Instruction35c
-import org.jf.dexlib2.iface.reference.StringReference
 import org.jf.dexlib2.iface.reference.TypeReference
 
 @Patch
 @DependsOn([IntegrationsPatch::class])
-@Name("settings")
+@Name("tiktok-settings")
 @Description("Adds ReVanced settings to TikTok.")
 @SettingsCompatibility
 @Version("0.0.1")
 class SettingsPatch : BytecodePatch(
     listOf(
+        AboutPageFingerprint,
         AdPersonalizationActivityOnCreateFingerprint,
         SettingsOnViewCreatedFingerprint,
     )
 ) {
     override fun execute(context: BytecodeContext): PatchResult {
-        SettingsOnViewCreatedFingerprint.result?.let {
-            AboutViewFingerprint.resolve(context, it.method, it.classDef)
-        }
-        // Patch Settings UI to add 'Revanced Settings'.
-        val targetIndexes = findOptionsOnClickIndex()
-        with(SettingsOnViewCreatedFingerprint.result!!.mutableMethod) {
-            for (index in targetIndexes) {
-                if (
-                    instruction(index).opcode != Opcode.NEW_INSTANCE ||
-                    instruction(index - 4).opcode != Opcode.MOVE_RESULT_OBJECT
-                )
+
+        AboutPageFingerprint.result?.let {
+            val startIndex = it.scanResult.patternScanResult!!.startIndex
+
+            with(it.mutableMethod.implementation!!.instructions) {
+                copyrightPolicyLabelId = (this[startIndex] as WideLiteralInstruction).wideLiteral
+            }
+
+        } ?: return AboutPageFingerprint.toErrorResult()
+
+        SettingsOnViewCreatedFingerprint.result?.mutableMethod?.let { method ->
+            val copyrightInstructions = method.implementation!!.instructions
+
+            val copyrightIndex = copyrightInstructions.indexOfFirst {
+                it.opcode == Opcode.CONST_STRING &&
+                        (it as BuilderInstruction21c).reference.toString() == "copyright_policy"
+            } - 6
```

Comment:
This likely breaks compatibility with older versions mentioned [here](https://github.com/revanced/revanced-patches/blob/d47f58bbce404bad70561995e752415c43a628db/src/main/kotlin/app/revanced/patches/tiktok/misc/settings/annotations/SettingsCompatibility.kt). Have you checked that?